client/blocks/signup-form/social.jsx

```diff
@@ -110,7 +117,7 @@ class SocialSignupForm extends Component {
 							responseHandler={ this.handleGoogleResponse }
 							uxMode={ uxMode }
 							redirectUri={ this.getRedirectUri( 'google' ) }
-							onClick={ () => this.trackSocialSignup( 'google' ) }
+							onClick={ this.trackLoginAndRememberRedirect.bind( null, 'google' ) }
```

Comment:
Is this change necessary? I thought that inlining an arrow function would bind `this` to the lexical scope 🤔  The same thing for the `onClick` prop below.